sphinx/conf.py

```diff
@@ -93,12 +93,13 @@
         }
     ],
     "use_edit_page_button": False,
+    "show_toc_level": 1,
     "collapse_navigation": True,
     "footer_items": ["copyright"],
     "left_sidebar_end": [],
     "logo": {
-      "image_light": "_static/BW_BlackandWhite_transparent.png",
```

Comment:
I updated the existing logo for landscape/horizontal display. Files have been added to the documentation `_static` directory as `svg` for better resolution. Compare my PR of the `logo` repo: https://github.com/brightway-lca/logo/pull/1